beacon/lighthouse/subscription.go

```diff
@@ -0,0 +1,64 @@
+package lighthouse
+
+import (
+	"time"
+
+	"github.com/prometheus/common/log"
```

Comment:
this is not the correct log package 
you should define a global (package) variable `var log = logrus.WithField("module", "lighthouse")` and use that one

Reply:
righty ho